_data/releases.yaml

```diff
@@ -9,7 +9,7 @@ operator:
     quickstart_book: true
     configuring_book: true
     deploying_book: true
-    directory_layout: layout3
+    directory_layout: layout4
```

Comment:
0.32 still has the Quickstart. The layout 4 should be used only for 0.33 o the future. So I think you should keep the layout, but revert this change.